omod-common/src/main/java/org/openmrs/module/webservices/rest/web/ConversionUtil.java

```diff
@@ -414,6 +417,24 @@ public static <S> Object convertToRepresentation(S o, Representation rep, Conver
 		}
 	}
 	
+	/**
+	 * Converts an object to its custom representation
+	 * This could be used to convert any domain objects that does not have any specific converter associated with them
+	 * such as SimpleObject's, Map's, etc
+	 */
+	private static SimpleObject convertToRepresentation(Object o, CustomRepresentation rep) {
```

Comment:
Is there any reason why the method name is generic yet from the description, you are dealing with only custom representations?

Reply:
I did thought about that a bit. I thought of overloading the method, to make it consistent with the rest of the methods.

If the concern is that, calls to `convertToRepresentation` resolving to a wrong method, we could refactor the method name. Since this method is private, there's no such thing from happening, though.

Would you like make it more specific like `convertToCustomRepresentation`? 

I don't have a preference on which one to use :)

Reply:
I refactored the method name.